File: ios/SpentyAI/SpentyAI/Core/Models/DashboardSummary.swift
import Foundation

struct DashboardSummary: Codable {
    var netWorth: Double?
    var incomeThisMonth: Double?
    var expenseThisMonth: Double?
    var pendingReview: Int?
    var totalAssets: Double?
    var totalLiabilities: Double?
    var savingsThisMonth: Double?
    var accounts: [Account]?
    var recentTransactions: [Transaction]?
}

File: android-native/app/src/main/java/com/spentyai/app/core/models/DashboardSummary.kt
package com.spentyai.app.core.models

import kotlinx.serialization.Serializable

@Serializable
data class DashboardSummary(
    val netWorth: Double? = null,
    val incomeThisMonth: Double? = null,
    val expenseThisMonth: Double? = null,
    val pendingReview: Int? = null,
    val totalAssets: Double? = null,
    val totalLiabilities: Double? = null,
    val savingsThisMonth: Double? = null,
    val accounts: List<Account>? = null,
    val recentTransactions: List<Transaction>? = null
)
